assets/monitoring-plugin/deployment.yaml

```diff
@@ -45,17 +45,12 @@ spec:
             topologyKey: kubernetes.io/hostname
       automountServiceAccountToken: false
       containers:
-      - command:
-        - /bin/sh
-        - -c
-        - |
-          if echo "$POD_IP" | grep -qE '^([0-9]{1,3}\.){3}[0-9]{1,3}$'; then
-            LISTEN_ADDRESS_PORT_REPLACED_AT_RUNTIME="9443"
-          else
-            LISTEN_ADDRESS_PORT_REPLACED_AT_RUNTIME="[::]:9443"
-          fi
-          sed "s/LISTEN_ADDRESS_PORT_REPLACED_AT_RUNTIME/$LISTEN_ADDRESS_PORT_REPLACED_AT_RUNTIME/g" /etc/nginx/nginx.conf > /tmp/nginx.conf
-          exec nginx -c /tmp/nginx.conf -g 'daemon off;'
+      - args:
+        - -static-path=/opt/app-root/web/dist
+        - -cert=/var/cert/tls.crt
+        - -key=/var/cert/tls.key
+        command:
+        - /opt/app-root/plugin-backend
         env:
         - name: POD_IP
```

Comment:
`POD_IP` was only used above, we can delete it now.

Reply:
I removed the POD_IP from the `deployment.yaml` and the `monitoring-plugin.libsonnet` in the most recent commit